src/test/lib/DataTest.cpp

```diff
@@ -788,6 +788,117 @@ QuicTestClientDisconnect(
     }
 }
 
+void
+QuicTestStatelessResetKey(
+    bool ChangeStatelessResetKey
+    )
```

Comment:
```suggestion
void
QuicTestStatelessResetKey(
    )
```
Just don't pass the parameter and always change the key.